src/components/common/needsShutdown.tsx

```diff
@@ -152,6 +197,14 @@ export function getDevicesRequiringShutdown(vm: VM) {
     if (needsShutdownSpice(vm))
         devices.push(_("SPICE"));
 
+    // VNC
+    if (needsShutdownVnc(vm))
+        devices.push(_("VNC"));
+
+    // Serial console
+    if (needsShutdownSerialConsole(vm))
+        devices.push(_("Text console"));
```

Comment:
"Text" leftover

Reply:
Yes, and a lot more... jumped the push gun...